src/tooltip/src/TooltipStateless.js

```diff
@@ -2,13 +2,14 @@ import React, { memo, forwardRef } from 'react'
 import PropTypes from 'prop-types'
 import { Pane } from '../../layers'
 import { Paragraph } from '../../typography'
-import useTooltipStyle from '../../theme/src/hooks/useTooltipStyle'
+import useStyleConfig from '../../hooks/use-style-config'
 
 const TooltipStateless = memo(
   forwardRef(function TooltipStateless(props, ref) {
     const { children, appearance, ...restProps } = props
-    console.log(appearance)
-    const { color, ...themedProps } = useTooltipStyle(appearance)
+    const { ...boxProps } = useStyleConfig('Tooltip', { appearance }, {}, {})
```

Comment:
This doesn't need to be spread
```suggestion
    const styleProps = useStyleConfig('Tooltip', { appearance }, {}, {})
```

Also can you make those empty objects consistent:
```js
const pseudoSelectors = {}
const internalStyles = {}
```

(We could also make them optional and have this behavior inside `useStyleConfig` to preserve stable object references and reduce hook reruns).

Reply:
+1 to making these optional, but out of scope, for now :)